valhalla/baldr/graphreader.h

```diff
@@ -695,6 +695,13 @@ class GraphReader {
     return edgeinfo(edgeid, NO_TILE);
   }
 
+  /**
+   * Get the shape of an edge
+   * @param edgeid
+   * @return the encoded shape (string) for specified id.
+   */
+  std::string edge_shape(const valhalla::baldr::GraphId& edgeid);
```

Comment:
this shoiuld probably be named such that you can tell it returns the polyline "encoded_edge_shape"